{{cookiecutter.project_name}}/databricks-config/prod/training-job.tf

```diff
@@ -13,7 +13,7 @@ resource "databricks_job" "model_training_job" {
   task {
     task_key = "Train"
 
-    {% if cookiecutter.include_feature_store %}notebook_task {
+    {% if cookiecutter.include_feature_store == "yes" %}notebook_task {
```

Comment:
Nit: we could actually in the `pre_gen_project.py` hook convert the "yes" or "no" input to this parameter into a boolean, so that we can just do the boolean check in project files

Reply:
Good idea, done: https://github.com/databricks/mlops-stack/pull/19/commits/a84cc7927c2a81b585899089118c28a20eb2c81f

Reply:
Hmm, this doesn't completely work, so I reverted the commit. It gets a bit messy and I think the main reason is that this version of cookiecutter doesn't support boolean variables. Let's clean this up once we upgrade cookiecutter to 2.2.0 (w/ proper boolean vars).